runtimes/core/src/model/mod.rs

```diff
@@ -288,3 +288,55 @@ pub struct AuthSuccessResponse {
     /// The user data.
     pub user_data: serde_json::Map<String, serde_json::Value>,
 }
+
+// matches go runtime
+pub enum LogLevel {
```

Comment:
In the logging overhaul PR I removed the log level enum we had and relied on `::log::Level` instead. Maybe better to do so here, too?